vimrcs/basic.vim

```diff
@@ -59,11 +59,8 @@ command! W execute 'w !sudo tee % > /dev/null' <bar> edit!
 " Set 7 lines to the cursor - when moving vertically using j/k
 set so=7
 
-" Avoid garbled characters in Chinese language windows OS
-let $LANG='en' 
-set langmenu=en
```

Comment:
Delete these two lines as I found we can safely use localized menu languages, as long as we reload menu **after** properly set fileencodings. 
There is no need to explicitly set $LANG and langmenu here. Vim will try to use system language.